modules/cnv-about-upgrading-cnv.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/upgrading-container-native-virtualization.adoc
+
+[id="cnv-about-upgrading-cnv_{context}"]
+= About upgrading {CNVProductName}
+
+If you enabled automatic updates when you installed {CNVProductName}, you
+will receive updates as they become available.
+
+.Additional information
+
+* In {CNVProductName} version 2.1, only _z-stream_ updates are available.
+** For example: {CNVProductName} 2.1.0 -> {CNVProductName} 2.1.1
+
+* Updates are delivered via the _Marketplace Operator_, which is deployed
+during {product-title} installation. The Marketplace Operator makes
+external Operators available to your cluster.
+
+* Upgrading does not interrupt virtual machine workloads.
+** Because workloads are uninterrupted, virtual machine Pods are not restarted
+or migrated during an upgrade. If you need to update the `virt-launcher` Pod,
```

Comment:
Just curious, do people know what the `virt-launcher` pod is (the pod which actually runs the VMI)?
But I also have no better idea to describe this :)

Reply:
I agree it's a good idea to add a little more context here, so I'll try to include your brief description. Thanks for catching that!

Reply:
@slintes Do you think this is accurate?
"Each virtual machine has a `virt-launcher` Pod that runs the virtual machine
instance. The `virt-launcher` Pod contains an instance of `libvirt`, which is
used to manage the virtual machine process."

Reply:
I'd prefer `The virt-launcher Pod *runs* an instance...`, otherwise :+1:

Reply:
Sounds good, I'll make that change now.

Reply:
Not sure if I understand...?
IMHO it's clear that "uninterrupted" is in the context of upgrades only.
But removing that part also works.
I don't have a preference.

Reply:
@alexxa I like your suggestion to remove "Because workloads are uninterrupted." I think the phrase is somewhat confusing in the context of that sentence, and the intended meaning is already conveyed through the bullet point levels.